internal/cli/cli.go

```diff
@@ -0,0 +1,49 @@
+package cli
+
+import (
+	"fmt"
+)
+
+const (
+	Reset  = "\033[0m"
+	Red    = "\033[31m"
+	Green  = "\033[32m"
+	Yellow = "\033[33m"
+	Blue   = "\033[34m"
+	Purple = "\033[35m"
+	Cyan   = "\033[36m"
+	Gray   = "\033[37m"
+	White  = "\033[97m"
+)
+
+const banner = `
```

Comment:
It looks fantastic! Could you please share how you created this banner? Additionally, I'd like to point out that there's a possibility that some fonts used might not be under the Apache license.

Reply:
Comment added.